status-im/status-react
# [ISSUE #2642] Allow to send ERC20 tokens

fixes #2642 
fixes #2645 
fixes #2009 

### Summary:

Allow to send ERC20 tokens. Also improves QR code handling (so that token request can be encapsulated) and allow to define gas details.

### Steps to test:
- Open Status
- Navigate to wallet
- Send tokens, it should work
- Optionally modify gas details

status: ready
